stl/inc/__msvc_iter_core.hpp

```diff
@@ -456,6 +456,9 @@ _EXPORT_STD using ranges::get;
 template <class _It, class _Se, ranges::subrange_kind _Ki>
 constexpr bool _Is_subrange_v<ranges::subrange<_It, _Se, _Ki>> = true;
 
+template <class _It, class _Se, ranges::subrange_kind _Ki>
+constexpr bool _Tuple_like_impl<ranges::subrange<_It, _Se, _Ki>> = true;

```

Comment:
```
D:\build\out\inc\__msvc_iter_core.hpp(460,16): error: no variable template matches partial specialization
  460 | constexpr bool _Tuple_like_impl<ranges::subrange<_It, _Se, _Ki>> = true;
      |                ^
```

It appears that the primary template is C++23, but this partial specialization is guarded for C++20:

https://github.com/microsoft/STL/blob/62205ab155d093e71dd9588a78f02c5396c3c14b/stl/inc/utility#L212-L215

Reply:
"Surely I must have tested these changes from months ago." :roll_eyes: